### ISSUE TITLE
Again, on fixing mobile usability issues

### ISSUE THREAD

**Issue by phronimos** [google/WebFundamentals#1174]
At 4:05pm on 3/22/2015, I sent my 2nd e-mail comment on fixing mobile usability issues to 

https://github.com/google/WebFundamentals/issues/1173

only to have it disappear into cyberspace because issue #1173 was summarily closed about 2:00pm.

Given that today is a Sunday, it never occurred to me that we would only be allowed about 6 hours for discussion, or I would have responded sooner (anything rather than have to open my own issue! ;-).

Because I do not feel the issues I have already raised about designing for the content -- to quote the guide again:

	"Content is the most important aspect of any site. So let’s
	design for the content and not let the design dictate the
	content."

-- have been adequately addressed, I thought I'd try raising them one more time here.

IMO, designing for the content takes precedence over "cater[ing] for users across all form factors."

And for me, this isn't simply a matter of style: it's about my ability to create high-quality, scholarly content, where website design has a didactic function which apparently places it at odds with "mobile-friendly" websites, as determined by Google Search.

That is the point I was trying to make earlier, with the response to PaulKinlan I was locked out from posting to

https://github.com/google/WebFundamentals/issues/1173#issuecomment-84682733

this afternoon, so am posting again now:

---------------------------------------------------------------------------------------------------
On 3/22/2015 12:26 PM, Paul Kinlan wrote:
> this is the problem, the lack of understanding of why
> it is important to cater for users across all form factors.
> You are forcing users to zoom in on your content on a
> mobile device.  Google search has always been about
> giving users the best result at the best time, and if
> users can't easily read your content, it is not the best
> result.

Just as I thought. Google Search is now evaluating content based on whether or not users can "easily read" it.

But this is not the goal -- nor should it be -- of publishing postdoctoral research.

Moreover, readability standards vary for different audiences, as I have documented here:

http://she-philosopher.org/home/design_2W1.html

So on what grounds does Google presume to decide what I -- a person, not a statistic -- will find readable?

... What trade-offs I am willing to make at a particular point in time -- and those trade-offs will vary a lot depending on ever-changing circumstance and context -- in order to access educational content that I believe might be of value to me?

> Un-happy users means less time on site,
> less click's through to other content
> and less chance they will convert
> (if you take payments).

Again, I fail to see how it is Google Search's mission to predetermine whether or not my website will make visitors "happy" enough to stick around and read my content.

As bobgasink argued, that negotiation should not be mediated by Google, but should be worked out directly between the visitors and the website developer(s).

Instead, Google Search is using its near monopoly on search to push a business model that's proven profitable for Google, but is irrelevant to scholarly sites like mine, which will not accept advertising.

> Just to reiterate, if you don't
> want to make your content work well
> on mobile, that is perfectly fine.

I didn't say that.

What I said was I don't want Google Search determining how I should or should not design content that it hasn't even indexed and doesn't understand.

I get that

    "Google search has always been about giving users the best
    result at the best time"

as long as advertising revenues drive search results and rankings.

But it is simply *not* true that Google gives users the best "content" available on the Web in response to every query it receives.

In fact, the new design standards require me to dumb down my content for some statistical mass of users who are probably not my target audience to begin with.

And when I won't do this, Google Search is going to bury my content and warn off potential visitors who may well be in my target audience, but who are intimidated enough by technology and/or Google's imprimatur that they worry my website is going to somehow harm their phones, or isn't safe to visit.

If you're truly concerned about giving users "the best result at the best time," how about giving them meaningful information, such as:

    This site will require you to pan around and zoom in order to
    read its content on your mobile phone.

?

In other words, tell them what the technical issues truly are, and let them make up their own minds about what kind of content they value, and how happy it makes them.

Deborah
---------------------------------------------------------------------------------------------------

I probably should state for the record that I will not be able to engage anyone further on this issue until late tomorrow (Monday).

And if such a leisurely style of deliberation isn't permitted at GitHub, so be it. I expect I've made my point by now.

**Comment by PaulKinlan** [comment]
Sorry, this isn't a discussion forum to decide on whether you should go mobile or not.  This is an Issue tracker to deal with issues in the guidance of the Web Fundamental project that helps you once you've made that decision.

With regards to your specific issues, I subscribe to the Marshall McLuhan thought in "The Medium Is the Massage."[sic] which points towards the content, the method of delivery of that content and how it is consumed to be critical.  In this case, by not optimizing for the medium in which the user reads your content then you are actively insulting them and saying that their needs are not as important as yours.

>  In fact, the new design standards require me to dumb down my content for some statistical mass of users who are probably not my target audience to begin with.

Again your are demonstrating your ignorance of the medium, the user and building for the web.  You don't have to dumb down a single thing to target mobile.

> In other words, tell them what the technical issues truly are, and let them make up their own minds about what kind of content they value, and how happy it makes them.

We have done this in the recent past and feedback was that it should also be considered a ranking factor.
